Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/AddBasket-AddNonprofitsViewController.m

```diff
@@ -62,6 +74,12 @@ - (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView cellForR
     cell.nonprofitNameLabel.text = nonprofit.nonprofitName;
     cell.nonprofitDescriptionLabel.text = nonprofit.nonprofitDescription;
     cell.nonprofitProfileImageView.image = [Utils getImageFromPFFile:nonprofit.headerPicFile];
+    
+    NSArray<NSIndexPath *> *selectedIndexes = [tableView indexPathsForSelectedRows];
+    BOOL isSelected = selectedIndexes != nil && [selectedIndexes containsObject:indexPath];
+    if (isSelected) {
+        cell.contentView.backgroundColor = [UIColor blueColor];
```

Comment:
You can use the `selectionStyle` property as I recommended earlier, and here you would just set `cell.isSelected = YES`.